<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Select</class>
 <widget class="QWidget" name="Select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择界面</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/build/Desktop_Qt_6_7_0_MinGW_64_bit-Debug/resource/icon/猫咪.svg</normaloff>:/icon/build/Desktop_Qt_6_7_0_MinGW_64_bit-Debug/resource/icon/猫咪.svg</iconset>
  </property>
  <widget class="QLabel" name="lab_player1_show">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>380</width>
     <height>620</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_player2_show">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>380</width>
     <height>620</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_player1_last">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>640</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>上一个角色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_player1_next">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>640</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>下一个角色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_player2_last">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>640</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>上一个角色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_player2_next">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>640</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>下一个角色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_over">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1380</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_show_is_ai">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>90</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_return_home">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>260</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>返回首页</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>lab_player1_show</zorder>
  <zorder>lab_player2_show</zorder>
  <zorder>btn_player1_last</zorder>
  <zorder>btn_player1_next</zorder>
  <zorder>btn_player2_last</zorder>
  <zorder>btn_player2_next</zorder>
  <zorder>btn_start</zorder>
  <zorder>btn_over</zorder>
  <zorder>lab_show_is_ai</zorder>
  <zorder>btn_return_home</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
